exchange/src/exchange/rabbitmq/publisher/RabbitMQPublisher.cpp

```diff
@@ -61,22 +58,18 @@ RabbitMQPublisher::broadcast_ob_updates(
     const std::vector<messages::ObUpdate>& updates, const std::string& ignore_uid
 )
 {
-    auto broadcast_to_client = [&](auto&& trader) {
-        if (!trader.is_active() || trader.get_id() == ignore_uid) {
+    const auto& traders = clients.get_traders();
+    for (const auto& [id, trader] : traders) {
+        if (!trader->is_active() || id == ignore_uid) {
```

Comment:
steev you're doing `strcmp`s and you worry about the overhead of a virtual function call?